src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int index = (random.nextInt(Color.values().length));
+        return Color.values()[index].name();
```

Comment:
Instead of calling `Color.values()` twice, which creates a new array each time, you can call it once, store it in a variable, and then use the variable. This will improve performance.